.clang-format

```diff
@@ -19,3 +19,24 @@ AllowShortBlocksOnASingleLine: 'Never'
 AlignArrayOfStructures: Left
 IndentCaseLabels: true
 AllowShortEnumsOnASingleLine: false
+---
+Language:  C
```

Comment:
This seems unrelated?

Reply:
Right, I need to use that to be able to run clang-format. Accidentally committed it.